### DISCUSSION TITLE
move_urlbar.uc.js not working in FF Developer 100

### DISCUSSION THREAD

**Discussion by Delfond** [Aris-t2/CustomJSforFx#68] [General]
The move_urlbar.uc.js is not working in firefox developer 100.

It also does not appear to work in conjunction with the additional_top_toolbars.uc.js. You cannot move the urlbar to an additional toolbar.

**Comment by tortious** [comment]
Pretty sure this one hasn't worked for a long time.

**Comment by Aris-t2** [comment]
Still work for me on Nightly 102 by just changing

var backbutton_on_toolbar =   
var forwardbutton_on_toolbar = 
var urlbar_on_toolbar = 

![image](https://user-images.githubusercontent.com/6013569/166479770-adbc0b4c-4166-453f-a006-c625e5283a2e.png)
![image](https://user-images.githubusercontent.com/6013569/166479964-21e238d4-1c3f-4d87-aef9-448f8aa9072f.png)
![image](https://user-images.githubusercontent.com/6013569/166480133-6e6b52f1-bc90-4254-8ab8-daebd35a7428.png)


You are right about the compatibility to additional_top_toolbars.uc.js. It does not work, as this are no official toolbars from Mozillas CustomizableUI.jsm module.
